salt/modules/x509.py

```diff
@@ -1366,3 +1366,81 @@ def verify_crl(crl, cert):
         return True
     else:
         return False
+
+
+def expired(certificate):
+    '''
+    Returns a dict containing limited details of a
+    certificate and whether the certificate has expired.
+
```

Comment:
Can you add a `.. versionadded:: Carbon` directive here, as well as in the new function below?